x-pack/plugin/core/template-resources/src/main/resources/logs-otel@mappings.json

```diff
@@ -0,0 +1,60 @@
+{
+  "template": {
+    "mappings": {
+      "date_detection": false,
+      "dynamic": "strict",
+      "properties": {
+        "data_stream.type": {
+          "type": "constant_keyword",
+          "value": "logs"
+        },
+        "observed_timestamp": {
+          "type": "date_nanos"
+        },
+        "severity_number": {
+          "type": "byte"
+        },
+        "severity_text": {
+          "type": "keyword"
+        },
+        "log.level": {
+          "type": "alias",
+          "path": "severity_text"
+        },
+        "body_text": {
```

Comment:
It would be nice if this would instead be based on https://github.com/elastic/elasticsearch/issues/109954 so both are the originals ...

Reply:
For these top-level fields, there should be a single source of truth. In the ES exporter, we have complete control over the document structure, and there should not be any custom fields at the root of the document. Only attributes contain dynamic parts.